docs/site/Migrating-from-LoopBack-3.md

```diff
@@ -0,0 +1,160 @@
+---
+lang: en
+title: 'Migrating from LoopBack 3'
+keywords: LoopBack 4.0, LoopBack 4, LoopBack 3
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Migrating-from-LoopBack-3.html
+---
+
+For current LoopBack 3 users who want to migrate to LoopBack 4, LoopBack 4
+offers a way to mount your LoopBack 3 application in a LoopBack 4 project. By
+adding your application this way, the application's REST API is included in the
+OpenAPI spec provided by the LoopBack 4 application. This also means that the
+LoopBack 3 application's models can be used with the LoopBack 4 REST API
+Explorer.
+
+## Mounting the LoopBack 3 Application in a LoopBack 4 Project
+
+{% include note.html content="
+If you are not familiar with LoopBack 4, visit
+[Getting started](Getting-started.md) for an introduction.
+" %}
+
+1. To mount your LoopBack 3 application, you will need to first create a new
+   LoopBack 4 application.
+
+   ```sh
+   $ lb4 app
```

Comment:
There are a lot of questions posed by this command, what do you suggest they specify?